pkg/app/sensor/app.go

```diff
@@ -477,15 +478,26 @@ func getCurrentPaths(root string) (map[string]interface{}, error) {
 				return filepath.SkipDir
 			}
 
-			if info.Mode().IsRegular() &&
-				!strings.HasPrefix(pth, "/proc/") &&
```

Comment:
we should keep explicit filters for /proc, /sys and /dev even if those paths are filtered another way